### PR TITLE
designed and made a fully functional signup UI view:rocket:

### PR DESCRIPTION
**Pull request by Segun-Ade**
I designed a signup UI view based on the wireframe design.
This is a work in progress

### CODE REVIEW

File: src/components/signup/components/Signup.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/jsx-filename-extension */
```
Comment (justiceotuya) [inline src/components/signup/components/Signup.test.js]: There is no need to disable this... you are writing jsx so the file ought to be jsx. please change the file extension to jsx. remember if at any point your write a jsx syntax, then that file extension should be jsx

File: src/components/signup/sagas.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { put, takeEvery } from 'redux-saga/effects';
+import actionTypes from './actionTypes';
+import { registrationError, registrationSuccess } from './actions';
+import apiRequest from './utils';
+// import { STRINGS } from './constants';
```
Comment (justiceotuya) [inline src/components/signup/sagas.js]: remove all unused code

File: src/components/signup/components/Signup.test.js (hunk 1 above)
Comment (Segun-Ade) [reply src/components/signup/components/Signup.test.js]: Alright, thank you @justiceotuya

**Comment by enyeInc** [comment]
@Segun-Ade This is another PR that is over 250 lines of change. Break it down. You already know this expectation

**Comment by Segun-Ade** [comment]
@enyeInc the PR has been broken down

**Pull request merged**